src/renderers/shaders/ShaderChunk/shadowmap_pars_fragment.glsl

```diff
@@ -16,4 +16,124 @@
 
 	}
 
+	#if defined(POINT_LIGHT_SHADOWS)
+
+		float unpack1K ( vec4 color ) {
+		
+			const vec4 bitSh = vec4( 1.0 / (256.0 * 256.0 * 256.0), 1.0 / (256.0 * 256.0), 1.0 / 256.0, 1.0 );
+			return dot( color, bitSh ) * 1000.0;
+			
+		}
+
+		/**
+		*  cubeToUV() maps a 3D direction vector suitable for cube texture mapping to a 2D
+		*  vector suitable for 2D texture mapping. This code uses the following layout for the
+		*  2D texture:		
+		*  
+		*  xzXZ
+		*   y Y
+		*
+		*  Y - Positive y direction
+		*  y - Negative y direction
+		*  X - Positive x direction
+		*  x - Negative x direction
+		*  Z - Positive z direction
+		*  z - Negative z direction
+		*
+		*  Alternate code for a horizontal cross layout can be found here:
+		*  https://gist.github.com/tschw/da10c43c467ce8afd0c4
+		*/
+
+		vec2 cubeToUV( vec3 v, float texelSizeX, float texelSizeY ) {
+
+			// Number of texels to avoid at the edge of each square
+
+			vec3 absV = abs( v );
+
+			// Intersect unit cube
+
+			float scaleToCube = 1.0 / max( absV.x, max( absV.y, absV.z ) );
+			absV *= scaleToCube;
+
+			// Apply scale to avoid seams
+
+			// two texels less per square (one texel will do for NEAREST)
+			v *= scaleToCube * ( 1.0 - 4.0 * texelSizeY );
```

Comment:
@mkkellogg 

The new version (updated yesterday, before my post) reads:
```javascript
// two texels less per square (one texel will do for NEAREST)
vec2 zoom = 1.0 - 2.0 * vec2( 4.0, 3.0 ) * texelSize;
```
Did you intentionally use the older one because there were artifacts? Or did you just make the changes before the update?